sale_product_seasonality/models/sale_order.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2021 Camptocamp SA
+# @author: Julien Coux <julien.coux@camptocamp.com>
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+from ..utils import roundTime
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    # TODO: any better name for this field?
+    date_end = fields.Datetime()
```

Comment:
Maybe `commitment_date_end` as we use `commitment_date` as start date?
```suggestion
    commitment_date_end = fields.Datetime()
```